wgpu-hal/src/vulkan/instance.rs

```diff
@@ -595,7 +569,7 @@ impl crate::Instance for super::Instance {
         })?;
         let version = {
             profiling::scope!("vkEnumerateInstanceVersion");
-            entry.try_enumerate_instance_version()
+            unsafe { entry.try_enumerate_instance_version() }
```

Comment:
What are the safety requirements here?

Reply:
Same as any other Vulkan call through `ash`. They just didn't happen to be marked `unsafe` before.